apps/wallet/tests/balanceChanges.spec.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { expect, test } from './fixtures';
+import { createWallet } from './utils/auth';
+import { generateAddress, generateKeypairFromMnemonic } from './utils/localnet';
+
+const mnemonic = [
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+];
+
+test('send 300 SUI and view transaction activity', async ({
+    page,
+    extensionUrl,
+}) => {
+    const keypair = await generateKeypairFromMnemonic(mnemonic.join(' '));
+    const address = generateAddress(keypair);
+
+    await createWallet(page, extensionUrl);
+
+    await page.getByTestId('faucet-request-button').click();
+    await expect(page.getByTestId('coin-balance')).toHaveText('1,000SUI');
```

Comment:
I wouldn't depend on the exact value, because that makes these test non-repeatable in a given network (unless you use a random mnemonic). Instead you should probably validate that this values changes